pkg/prometheus/server/statefulset_test.go

```diff
@@ -1288,9 +1174,7 @@ func TestThanosObjectStorageFile(t *testing.T) {
 				}
 			}
 		}
-		if !found {
-			t.Fatalf("--tsdb.path argument should be given to the Thanos sidecar, got %q", strings.Join(sset.Spec.Template.Spec.Containers[3].Args, " "))
-		}
+		require.True(t, found, "--tsdb.path argument should be given to the Thanos sidecar, got %q", strings.Join(sset.Spec.Template.Spec.Containers[2].Args, " "))
```

Comment:
Same here too

> I changed the index from 3 to 2 because I received the following error locally on running the test
> 
> ```
> --- FAIL: TestThanosObjectStorageFile (0.00s)
> panic: runtime error: index out of range [3] with length 3 [recovered]
>         panic: runtime error: index out of range [3] with length 3
> ```
> After changing, it was running fine.